### DISCUSSION TITLE
Blog Issues/Documentation

### DISCUSSION THREAD

**Discussion by nacoleman** [abpframework/abp#6449] [Q&A]
I have just purchased ABP Commercial and have a test application up and running. I added the blogging module via ABP Suite and I can see I can now create blogs from within my application. However that is all i can do... 
- I cant seem to find the blog i just created 
- There is nowhere that allows me to create pages etc
- I cant find any documentation on the Blog and how to use it
- Is the blog multi-tenant? I can see a tenant Id against the blog entry in the DB

Thanks

**Comment by maliming** [comment]
hi

There are some page routes.

https://github.com/abpframework/abp/blob/47d180cf8e7b54ffb1524f62ebd087ea19b5a7a0/modules/blogging/src/Volo.Blogging.Web/BloggingWebModule.cs#L70-L73
